grpc/README.md

```diff
@@ -2,6 +2,18 @@
 
 This module holds the functionality for the gRPC library integrated into Aries. The implementation is currently focused towards [**unified_planning**](https://github.com/aiplan4eu/unified-planning).
 
+<!-- TODO: Update README for UP Server and Usage Information -->
+
+## GRPC API
```

Comment:
Just need to be a little more specific: generating the rust bindings (file XXXX) from the protobuf format definition (file XXX).